frontend/public/tinymce/langs/zh_TW.js

```diff
@@ -0,0 +1,419 @@
+"Error": "\u932f\u8aa4",
+"Warn": "\u8b66\u544a",
+"Valid": "\u6709\u6548",
+"To open the popup, press Shift+Enter": "\u8981\u958b\u555f\u5f48\u51fa\u8996\u7a97\uff0c\u8acb\u6309Shift+Enter",
+"Rich Text Area. Press ALT-0 for help.": "\u5bcc\u6587\u672c\u5340\u57df\u3002\u8acb\u6309ALT-0\u5c0b\u6c42\u5354\u52a9\u3002",
+"System Font": "\u7cfb\u7d71\u5b57\u578b",
+"Failed to upload image: {0}": "\u7121\u6cd5\u4e0a\u50b3\u5f71\u50cf\uff1a{0}",
+"Failed to load plugin: {0} from url {1}": "\u7121\u6cd5\u4e0a\u50b3\u63d2\u4ef6\uff1a{0}\u81eaurl{1}",
+"Failed to load plugin url: {0}": "\u7121\u6cd5\u4e0a\u50b3\u63d2\u4ef6\uff1a{0}",
+"Failed to initialize plugin: {0}": "\u7121\u6cd5\u555f\u52d5\u63d2\u4ef6\uff1a{0}",
+"example": "\u7bc4\u4f8b",
+"Search": "\u641c\u7d22",
+"All": "\u5168\u90e8",
+"Currency": "\u8ca8\u5e63",
+"Text": "\u6587\u672c",
+"Quotations": "\u5f15\u7528",
+"Mathematical": "\u6578\u5b78",
+"Extended Latin": "\u62c9\u4e01\u5b57\u6bcd\u64f4\u5145",
+"Symbols": "\u7b26\u865f",
+"Arrows": "\u7bad\u982d",
+"User Defined": "\u4f7f\u7528\u8005\u5df2\u5b9a\u7fa9",
+"dollar sign": "\u7f8e\u5143\u7b26\u865f",
+"currency sign": "\u8ca8\u5e63\u7b26\u865f",
+"euro-currency sign": "\u6b50\u5143\u7b26\u865f",
+"colon sign": "\u79d1\u6717\u7b26\u865f",
+"cruzeiro sign": "\u514b\u9b6f\u8cfd\u7f85\u7b26\u865f",
+"french franc sign": "\u6cd5\u6717\u7b26\u865f",
+"lira sign": "\u91cc\u62c9\u7b26\u865f",
+"mill sign": "\u6587\u7b26\u865f",
+"naira sign": "\u5948\u62c9\u7b26\u865f",
+"peseta sign": "\u6bd4\u585e\u5854\u7b26\u865f",
+"rupee sign": "\u76e7\u6bd4\u7b26\u865f",
+"won sign": "\u97d3\u571c\u7b26\u865f",
+"new sheqel sign": "\u65b0\u8b1d\u514b\u723e\u7b26\u865f",
+"dong sign": "\u8d8a\u5357\u76fe\u7b26\u865f",
+"kip sign": "\u8001\u64be\u5e63\u7b26\u865f",
+"tugrik sign": "\u8499\u53e4\u5e63\u7b26\u865f",
+"drachma sign": "\u5fb7\u514b\u62c9\u99ac\u7b26\u865f",
+"german penny symbol": "\u5fb7\u570b\u5206\u7b26\u865f",
+"peso sign": "\u62ab\u7d22\u7b26\u865f",
+"guarani sign": "\u5df4\u62c9\u572d\u5e63\u7b26\u865f",
+"austral sign": "\u963f\u6839\u5ef7\u5e63\u7b26\u865f",
+"hryvnia sign": "\u70cf\u514b\u862d\u5e63\u7b26\u865f",
+"cedi sign": "\u8fe6\u7d0d\u5e63\u7b26\u865f",
+"livre tournois sign": "\u91cc\u5f17\u723e\u7b26\u865f",
+"spesmilo sign": "\u570b\u969b\u5e63\u7b26\u865f",
+"tenge sign": "\u54c8\u85a9\u514b\u5e63\u7b26\u865f",
+"indian rupee sign": "\u5370\u5ea6\u76e7\u6bd4\u7b26\u865f",
+"turkish lira sign": "\u571f\u8033\u5176\u91cc\u62c9\u7b26\u865f",
+"nordic mark sign": "\u5317\u6b50\u99ac\u514b\u7b26\u865f",
+"manat sign": "\u4e9e\u585e\u62dc\u7136\u5e63\u7b26\u865f",
+"ruble sign": "\u76e7\u5e03\u7b26\u865f",
+"yen character": "\u65e5\u5713\u7b26\u865f",
+"yuan character": "\u4eba\u6c11\u5e63\u7b26\u865f",
+"yuan character, in hong kong and taiwan": "\u6e2f\u5143\u8207\u53f0\u5e63\u7b26\u865f",
+"yen\/yuan character variant one": "\u65e5\u5713\/\u4eba\u6c11\u5e63\u7b26\u865f\u8b8a\u5316\u578b",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687804)
```suggestion
'yen\/yuan character variant one': "\u65e5\u5713\/\u4eba\u6c11\u5e63\u7b26\u865f\u8b8a\u5316\u578b",
```